src/worker-bindings.cc

```diff
@@ -154,6 +155,48 @@ const char* Worker::InitIsolate() {
   return NULL;
 }
 
+void Worker::WriteCodeCoverageReport() {
+  Locker locker(isolate_);
+  Isolate::Scope isolate_scope(isolate_);
+  Nan::HandleScope scope;
+
+#if NODE_VERSION_AT_LEAST(0, 11, 0)
+  Local<Context> context = Local<Context>::New(isolate_, context_);
+  Context::Scope context_scope(context);
+#else
+  Handle<Context> context = context_;
+  Context::Scope context_scope(context_);
+#endif
```

Comment:
Well, it turns out I need the `context` variable in the code below (later). What do you propose - should I create something like `ContextWithScope` and then retrieve the context handle via `contextWithScope.context`? Do you have any better suggestion?

```cpp
ContextWithScope ctx(this); // uses this->isolate_ and this->context_
// later
Local<Object> global(ctx.context->Global());

// or perhaps implement -> operator, so that I can write
Local<Object> global(ctx->Global());
```

@bnoordhuis Thoughts?